### PR TITLE
Revert "Small UI/UX tweaks for Model and About pages"

### PR DESCRIPTION
**Pull request by hnykda**
Reverts epidemics/covid#267

@rakyi I am rather reverting - the chart is very narrow, unfortunately. Is that really what you intended?
image
image

### CODE REVIEW

**Comment by hnykda** [comment]
Compared to
image

**Comment by rakyi** [comment]
The original (current) version is even more narrow on my screen.

image

The trick is to find a right width to aim for, I guess. I opted for setting `max-width` to be consistent across different screen sizes. Relative column widths (original approach) has it's own downside, as you can see.

Maybe we could pick better set of column classes or just pick bigger `max-width`. I just picked slightly bigger width (650px) than the original when displayed on my screen (~630px).

**Comment by hnykda** [comment]
That's weird then that it showed like it did on my monitor :thinking: . @wolverdude @jeromezng any opinion?

**Comment by rakyi** [comment]
It's not weird, it's exactly how Bootstrap columns are supposed to work. You just have bigger screen than me.

This is the important class list: `class="col-xs-11 col-sm-10 col-md-9 col-lg-8 col-xl-6"`. On both of our screens `col-xl-6` class applies, which translates to 50% width, which is 631px in my case and somewhat more in yours.

**Comment by rakyi** [comment]
I arbitrarily picked 650px in my PR to fix it for my screen size without changing the width too much, but it "broke" for you, since you expected the page to be wider. IMO we need to decide what is the optimal width of the page and then either pick a constant `max-width` rule or try to fix the `col-*` classes so it isn't too wide or narrow on each breakpoint.

Sorry to spend so much energy on it in the comments, but I feel like it was quite unclear and wanted to clear it up.

**Comment by hnykda** [comment]
Got it, sorry, I am not an expert here and understand now. Let's wait for Jerome and Jon to have some consensus on this.

File: src/server/templates/model.html [hunk 1]
```diff
@@ -46,9 +46,10 @@
 </div>
 
 <!-- Main model graph -->
-<div class="container main-wrapper">
-  <div class="row justify-content-center">
-    <div class="col model-content">
+<div class="main-wrapper">
+  <div class="row">
+    <div class="col"></div>
+    <div class="col-xs-11 col-sm-10 col-md-9 col-lg-8 col-xl-6">
```
Comment (jeromezng) [inline src/server/templates/model.html]: I'm curious why we're using col-xs-11 instead of col-xs-12?

**Comment by jeromezng** [comment]
> It's not weird, it's exactly how Bootstrap columns are supposed to work. You just have bigger screen than me.
> 
> This is the important class list: `class="col-xs-11 col-sm-10 col-md-9 col-lg-8 col-xl-6"`. On both of our screens `col-xl-6` class applies, which translates to 50% width, which is 631px in my case and somewhat more in yours.

Yes this is fine as long as we're consistent that these are the sizes we optimize for across all the pages.

**Comment by hnykda** [comment]
#300 seems to throw this all off I guess for now :grimacing:

**Pull request merged**